components/TField.vue

```diff
@@ -33,7 +33,7 @@
           :hide-label="hideLabel"
           :item="item"
           v-on="$attrs.listeners"
-          @input="(val) => $emit('input', set(val))"
+          @update="(val) => $emit('input', set(val))"
```

Comment:
This will emit `input` event, but later on you use `update` event. Are you sure it works?